HW_7/task34.py

```diff
@@ -0,0 +1,12 @@
+# Задача 34
+
+poem = input("Введите стихотворение: ")
+poem = poem.lower()
+vowels = ['а', 'е', 'ё', 'и', 'о', 'у', 'ы', 'э', 'ю', 'я']
```

Comment:
Можно просто использовать строку.
```suggestion
vowels = 'аяуюоеёэиы'
```